### PR TITLE
Adds a `build:prod` script 

### LINKED ISSUE #55: React: Use the Production Build
It looks like we build CE with the development version of react
https://reactjs.org/docs/optimizing-performance.html#use-the-production-build

At least this is the case on the demo page 
http://cloud-explorer.org/

### PR DESCRIPTION
**Pull request by JbIPS**
* Creates a production build, uglified and minimized :robot: 
* Simplifies the fonts system (thx npm 5)
* Makes it Windows compatible

Closes #55

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,7 +13,8 @@
     "test": "jest",
     "test:watch": "npm test -- --watch",
     "build": "npm run build:js && npm run build:sass",
-    "build:fonts": "mkdir -p dist/fonts && cp -R `node_modules`/font-awesome/fonts/* `node_modules`/npm-font-open-sans/fonts/* dist/fonts/",
+    "build:prod": "browserify --extension=jsx src/js/App.jsx -t [ babelify ] -g [ envify --NODE_ENV production ] -g uglifyify | uglifyjs --compress --mangle > dist/cloud-explorer.js",
```
Comment (lexoyo) [inline package.json]: Why mangle?

File: package.json [hunk 2]
```diff
@@ -13,7 +13,8 @@
...
+    "build:fonts": "cross-env mkdir -p dist/fonts && (cp -R ../font-awesome/fonts/* ../npm-font-open-sans/fonts/* dist/fonts/) || (cp -R node_modules/font-awesome/fonts/* node_modules/npm-font-open-sans/fonts/* dist/fonts/)",
```
Comment (lexoyo) [inline package.json]: Why would node modules always be in `..`? It can be in `./node_modules` right?

Comment (JbIPS) [reply package.json]: Hence the `||`. It first tries `..` in the case CE is used as a module and if it fails, tries `node_modules` for a standalone version.

File: package.json (hunk 1 above)
Comment (JbIPS) [reply package.json]: I used React recommendations for Browserify https://reactjs.org/docs/optimizing-performance.html#browserify

File: package.json (hunk 2 above)
Comment (lexoyo) [reply package.json]: Han... :man_facepalming:

File: package.json (hunk 1 above)
Comment (lexoyo) [reply package.json]: Ok then, if it works then :)

**Pull request merged**